tests/test_domains/test_domain_py_pyobject.py

```diff
@@ -390,3 +390,98 @@ def test_pydecoratormethod_signature(app):
 
     assert 'deco' in domain.objects
     assert domain.objects['deco'] == ('index', 'deco', 'method', False)
+
+
+def test_pycurrentmodule(app):
+    text = (".. py:module:: Other\n"
+            "\n"
+            ".. py:module:: Module\n"
+            ".. py:class:: A\n"
+            "\n"
+            "   .. py:method:: m1\n"
+            "   .. py:method:: m2\n"
+            "\n"
+            ".. py:currentmodule:: Other\n"
+            "\n"
+            ".. py:class:: B\n"
+            "\n"
+            "   .. py:method:: m3\n"
+            "   .. py:method:: m4\n")
+    domain = app.env.get_domain('py')
+    doctree = restructuredtext.parse(app, text)
+    print(doctree)
```

Comment:
Is this print necessary?

Reply:
It's not. Several of the existing tests contain a similar `print`. It's normally hidden by the test runner, but on failure it can be a huge help in debugging/correcting the `assert_node` call. But I can certainly remove it.

Reply:
No it's fine ! I wasn't looking directly at the other tests but if I know we have prints here and there. It was just to make sure that you didn't forget some debug statement.